app/serializers/api/v2/serializable_provider.rb

```diff
@@ -62,6 +62,7 @@ def enrichment_attribute(name, enrichment_name = name)
       enrichment_attribute :train_with_disability
 
       has_many :sites
+      has_many :accrediting_providers
```

Comment:
use another serializer to trim down db requests

e.g. `SerializableAccreditingProvider`